tests/test_plot.py

```diff
@@ -74,6 +74,56 @@ def test_line_plots(function, handsome_signal, handsome_signal_v2):
                      file_type, compare_output)
 
 
+@pytest.mark.parametrize('show_real_imag_abs_flag', [
+    'real', 'imag', 'abs'])
+def test_complex_time_plots(show_real_imag_abs_flag,
+                            handsome_complex_signal,
+                            handsome_complex_signal_v2):
+    """Test all line plots with default arguments and hold functionality."""
+    print(f"Testing: {plot.time.__name__}, show_real_imag_abs "
+          f"= {show_real_imag_abs_flag}")
+
+    # initial plot
+    filename = plot.time.__name__ + '_default'
```

Comment:
```suggestion
    filename = f'{plot.time.__name__}_{show_real_imag_abs_flag}_default'
```
otherwise the baseline will be overwirtten anytime